neo4j-to-neptune/src/main/java/com/amazonaws/services/neptune/ConvertCsv.java

```diff
@@ -35,8 +38,27 @@ public class ConvertCsv implements Runnable {
     @Once
     private File outputDirectory;
 
+    @Option(name = {"-df", "--dotenv-file"}, description = "Path to the Dotenv file for Neo4j AuraDB connection.")
```

Comment:
Missing update in docs for new arguments: https://github.com/Bit-Quill/amazon-neptune-tools/blob/8d397d294ab5e0dbdba1f178ff53ccb7b2c08bbe/neo4j-to-neptune/docs/convert-csv.md?plain=1#L6-L10

Also note that in those docs, optional arguments are in square brackets, and required ones are not. I'm not sure how you might best express the new `RequiredOnlyOne` paradigm here.